ios/Podfile

```diff
@@ -16,7 +16,6 @@ target 'ledgerlivemobile' do
     :hermes_enabled => false
   )
 
-  pod 'djinni_objc', :path => '../node_modules/@ledgerhq/react-native-ledger-core'
```

Comment:
i'm not sure why this was committed here.

Reply:
🤷‍♂️ I had to do some pod install after the last rebase and I guess that was added by one of the rebase ?

I'll just remove it if you think it shouldn't be here

Reply:
Done 👍